src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerConfirmService.java

```diff
@@ -54,26 +54,26 @@ public void configure(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
             .page("caseworkerConfirmService")
             .pageLabel("Confirm Service")
             .complex(CaseData::getDocuments)
-                .optional(CaseDocuments::getDocumentsUploadedOnConfirmService)
-                .done()
+            .optional(CaseDocuments::getDocumentsUploadedOnConfirmService)
+            .done()
             .label("applicantLabel", "Name of Applicant - ${applicant1FirstName} ${applicant1LastName}")
             .label("respondentLabel", "Name of Respondent - ${applicant2FirstName} ${applicant2LastName}")
             .complex(CaseData::getApplication)
-                .complex(Application::getSolicitorService)
-                    .mandatory(SolicitorService::getDateOfService)
-                    .mandatory(SolicitorService::getDocumentsServed)
-                    .mandatory(SolicitorService::getOnWhomServed)
-                    .mandatory(SolicitorService::getHowServed)
-                    .mandatory(SolicitorService::getServiceDetails,
-                        "solServiceHowServed=\"deliveredTo\" OR solServiceHowServed=\"postedTo\"")
-                    .mandatory(SolicitorService::getAddressServed)
-                    .mandatory(SolicitorService::getBeingThe)
-                    .mandatory(SolicitorService::getLocationServed)
-                    .mandatory(SolicitorService::getSpecifyLocationServed, "solServiceLocationServed=\"otherSpecify\"")
-                    .mandatoryWithLabel(SolicitorService::getServiceSotName, "Solicitor's/Applicant's name")
-                    .optional(SolicitorService::getServiceSotFirm)
-                    .mandatory(SolicitorService::getStatementOfTruth)
-                .done()
+            .complex(Application::getSolicitorService)
+            .mandatory(SolicitorService::getDateOfService)
+            .mandatory(SolicitorService::getDocumentsServed)
+            .mandatory(SolicitorService::getOnWhomServed)
+            .mandatory(SolicitorService::getHowServed)
+            .mandatory(SolicitorService::getServiceDetails,
+                "solServiceHowServed=\"deliveredTo\" OR solServiceHowServed=\"postedTo\"")
+            .mandatory(SolicitorService::getAddressServed)
+            .mandatory(SolicitorService::getBeingThe)
+            .mandatory(SolicitorService::getLocationServed)
+            .mandatory(SolicitorService::getSpecifyLocationServed, "solServiceLocationServed=\"otherSpecify\"")
+            .mandatoryWithLabel(SolicitorService::getServiceSotName, "Solicitor's/Applicant's name")
+            .optional(SolicitorService::getServiceSotFirm)
+            .mandatory(SolicitorService::getStatementOfTruth)
+            .done()
```

Comment:
formatting can be reverted :)

Reply:
fixed here https://github.com/hmcts/nfdiv-case-api/pull/2236/commits/54f83d312c330bf9f841073ba985d6511e0ef400